R/02-importData-queryData.R

```diff
@@ -477,7 +477,8 @@ gptServer <- function(id, autoCompleteList, isActiveTab) {
                        prompt_content = paste("Write an SQL query to", input$gptPrompt),
```

Comment:
Here we could improve the results with some context. See https://github.com/Pandora-IsoMemo/data-tools/pull/74#issuecomment-2035205321

But i think this would be another issue...